Application/Business/LevelServices/LevelServices.cs

```diff
@@ -28,14 +29,48 @@ public async Task<int> Count()
         return await _levelRepo.CountAsync();
     }
     public async Task<int> Count(Guid gameId)
-    { // Count levels in 1 game
+    {
         var levels = await _levelRepo.WhereAsync(l => l.GameId.Equals(gameId));
         return levels.Count();
     }
     public async Task Create(LevelEntity level)
     {
-       
+        //2 levels of a same game cannot have the same name:
+        if(await checkExistLevel(level) is not null)
```

Comment:
move this logic to checkExistLevel  too, rename it (if necessary). The name of the method must be PascalCase